src/validation/signupValidation.js

```diff
@@ -0,0 +1,16 @@
+import Check from '../utils/validator';
+import sendResult from '../utils/sendResult';
+
+const validateSignup = (req, res, next) => {
+  const valid = [
+    new Check({ email: req }).req().str().email()
+      .min(4),
+    new Check({ username: req }).req().str().min(2),
+    new Check({ password: req }).req().alphaNum().min(8),
+  ];
+  const invalid = valid.find((e) => e.error);
+  if (invalid) return sendResult(res, 400, invalid.error);
```

Comment:
`password not being alphanumeric` thats the quote from the story description consider removing `.alphaNum()`